train_cnn.py

```diff
@@ -0,0 +1,118 @@
+import sys
+
+import numpy as np
+import pandas as pd
+
+import torch
+from torch import Tensor
+import torchvision.datasets as datasets
+import torchvision.transforms as transforms
+
+sys.path.insert(1, '/home/khushi/Documents/simple-neural-network/models')
```

Comment:
Eh? Why?
Why not declare init files and import like a python package?

Reply:
> Please don't laugh
Well no. I don't like to demean anyone. Sorry If I hurt you.
Don't worry you can do this into a package. :) Ping me anytime you need help

Reply:
> Well no. I don't like to demean anyone. Sorry If I hurt you.

Haha, nothing like that. Maybe it's my inner instincts. LOL